sds_data_manager/lambda_code/SDSCode/pipeline_lambdas/scheduled_job.py

```diff
@@ -45,15 +48,140 @@ def scheduled_processing_event(session, events):
     processing_input_collection = ProcessingInputCollection(*processing_inputs)
 
     for job in triggered_jobs:
-        batch_starter.try_to_submit_job(
+        try_to_submit_job(
             session,
             job,
-            "20000101",
+            dt.datetime.now(),
```

Comment:
You are right on the border with your 6:00 and the UTC / local time jumping between dates, lets try and stick with just UTC if OK.
```suggestion
   dt.datetime.now(dt.timezone.utc).strftime("%Y%m%d")
```